services/nss-coverage/launch-worker.sh

```diff
@@ -75,9 +75,16 @@ function clone-corpus {
   if [[ ! -d "$name" ]]; then
     mkdir "$name"
     cd "$name"
-    retry-curl -O "$url"
-    unzip public.zip
+
+    # There may be no OSS-Fuzz corpus yet for new fuzz targets
+    code=$(retry-curl --no-fail -w "%{http_code}" -O "$url")
+    if [[ $code -eq 200 ]]; then
+      unzip public.zip
+    else
+      echo "[nss-coverage] WARNING: failed to clone corpus for $name"
```

Comment:
looks good, but you should print to stderr (eg. `echo "..." >&2`) and maybe add `$code` to the message so if we look we see 404 and not 5xx

Reply:
Done. Also removed `[nss-coverage]` since it falls a bit out of the other logging style. My intention was to make it easier to query for the log limited to the nss-coverage service, but I think that should also be possible in another way.